azure-pipelines.yml

```diff
@@ -0,0 +1,71 @@
+resources:
+- repo: self
+trigger:
+  batch: true
+  branches:
+    include:
+    - '*'
+jobs:
+- job: Tox27
+  displayName: 'Tox: Python 2.7'
+  condition: succeeded()
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+  - UsePythonVersion@0
+    displayName: 'Use Python 2.7'
+    inputs:
+      versionSpec: 2.7
+  - task: Bash@3
+    displayName: 'Run Tox'
+    inputs:
+      targetType: 'filePath'
+      filePath: scripts/ci/run_tox.sh
```

Comment:
This is where I'd override the TOXENV environment variable to inform tox it should only run in one spot. That would make this section:

``` yaml
- task: Bash@3
  displayName: 'Run Tox'
  env:
    TOXENV: py27
  inputs:
    targetType: 'filePath'
    filePath: scripts/ci/run_tox.sh
```
The other Python versions should follow suit as well, of course.

I'm sure you've already seen it, but the documentation I'm working from can be found here: [tox.readthedocs.io#selecting-one-or-more-environments](https://tox.readthedocs.io/en/latest/example/general.html#selecting-one-or-more-environments-to-run-tests-against)